PLEP-0007.rst

```diff
@@ -0,0 +1,262 @@
+| :directory: ``./plasmapy/formulary/``                                        |
+| :access: ``plasmapy.formulary``                                              |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A sub-package that provides mathematical and scientific formulas for       |
+|   calculating physical parameters of various plasmas.  This is inspired by,  |
+|   and akin to, the `NRL Plasma Formulary`_.                                  |
++------------------------------------------------------------------------------+
+| .. _particles:                                                               |
+|                                                                              |
+| **particles**                                                                |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/particles/``                                        |
+| :access: ``plasmapy.particles``                                              |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A sub-package that fully defines the properties of a "particle."  A        |
+|   "particle" can come in many forms ranging from a traditional particle      |
+|   (electron, ion, atom, etc.) to more exotic types like dust particles,      |
+|   dimensionless particles for simulations, super-particles for simulations,  |
+|   etc.                                                                       |
++------------------------------------------------------------------------------+
+| .. _plasma:                                                                  |
+|                                                                              |
+| **plasma**                                                                   |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/plasma/``                                           |
+| :access: ``plasmapy.plasma``                                                 |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A sub-package that fully defines a plasma.  This would include the         |
+|   plasma's species constituents and physical parameters (like temperature,   |
+|   boundary conditions, magnetic fields, etc.).                               |
+|                                                                              |
+|   Any tools that go into defining a plasma or its environment (e.g. a field  |
+|   solver) should be included in a sub-package within ``plasmapy.plasma``.    |
```

Comment:
Hm...I'm wondering about how to fit this in with classes to define initial conditions in `plasmapy.simulation`.   But, that's an issue for another day.

Reply:
yeah, there will probably be some cross over there.🤔